lib/lotus/commands/generate.rb

```diff
@@ -0,0 +1,43 @@
+require 'lotus/utils/string'
+require 'lotus/utils/class'
+
+module Lotus
+  module Commands
+    class Generate
+      GENERATORS_NAMESPACE = "Lotus::Generators::%s".freeze
+
+      class Error < ::StandardError
+      end
+
+      attr_reader :cli, :source, :target, :app, :app_name, :app_root, :name, :options
```

Comment:
`app_root` method is already defined below, defining reader for `app_root` seems unnecessary